guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -42,22 +42,68 @@ ifdef::katello[]
 ----
 # {package-manager} config-manager --set-enabled powertools
 ----
+
++
+. Enable the Katello module:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-manager} module enable katello:el8
+----
 endif::[]
-ifdef::foreman-el,katello[]
+
+=== In case module enablement fails
+
+If the module fails to enable, it can mean an incorrect module is enabled. List the enabled modules to find out which ones:
```

Comment:
Intro paragraph after the heading: 
"If the XY*) module fails to enable, it can mean an incorrect module is enabled." + a sentence in the spirit "In that case, you have to resolve dependencies manually as follows." (or something like that to summarize what the user has to do about it next)